src/main/java/org/sagebionetworks/web/server/servlet/SynapseClientImpl.java

```diff
@@ -2114,16 +2115,21 @@ public ArrayList<String[]> parseCsv(String csvPreviewText, char delimiter)
   @Override
   public String getDatasetScriptElementContent(String entityId)
     throws RestServiceException {
-    String plainTextWiki = CrawlFilter.getPlainTextWiki(entityId, this);
-    EntityBundleRequest bundleRequest = new EntityBundleRequest();
-    bundleRequest.setIncludeEntity(true);
-    bundleRequest.setIncludeAnnotations(true);
-    EntityBundle entityBundle = getEntityBundle(entityId, bundleRequest);
-    try {
-      return CrawlFilter.getDatasetScriptElementContent(
-        entityBundle,
-        plainTextWiki
-      );
+    try {
+      org.sagebionetworks.client.SynapseClient synapseClient = createSynapseClient();
+      Entity entity = synapseClient.getEntityById(entityId);
```

Comment:
Change here is to check to see if the entity ID points to a Dataset entity.  Only return element content if it is a Dataset.